source/except.d.ts

```diff
@@ -25,9 +25,18 @@ type Filtered = Filter<'bar', 'foo'>;
 //=> 'bar'
 ```
 
-@see {Except}
+@see {NonStrictExcept}
 */
-type Filter<KeyType, ExcludeType> = IsEqual<KeyType, ExcludeType> extends true ? never : (KeyType extends ExcludeType ? never : KeyType);
+type Filter<KeyType, ExcludeType> =
+IsEqual<KeyType, ExcludeType> extends true ? never :
+	KeyType extends ExcludeType ? never : KeyType;
+
+type ExceptOptions = {
+	/**
+	@default false
+	*/
+	strict?: boolean;
+};
```

Comment:
Could add a description to the option:

```suggestion
type ExceptOptions = {
	/**
	Disallow assigning non-specified properties.

    Setting this to `false` is not recommended. Included for backwards-compatability.

	@default false
	*/
	strict?: boolean;
};
```